### DISCUSSION TITLE
Junkers/Bosh   boiler accept only read vaules  not write

### DISCUSSION THREAD

**Discussion by matteopedani** [emsesp/EMS-ESP32#1064] [General]
Using a EMS-ESP32 with Junkers or Bosh  boiler only don't work.
You can read the values from the boiler, but you cant' write it any command  to the boiler like on/off, temperature. And the boiler is all time ON.

Looking in PDF  manual ,    if  the boiler sense EMS device, stop work from   ON/OFF switch control, and the boiler stay  ON. The manual describe, a setting you can  modify from auto to to ON/OFF device, but didn't work for me. 

I think is the boiler aspect a boot sequence   commands came from thermostat  and the EMS-ESP32 don't generate  them. 

There is a person who give as the log of boot  between one boiler and one or more thermostats? I have only a boiler!

**Comment by proddy** [comment]
Can you look at https://emsesp.github.io/docs/Support/ and attach the system details? I haven't seen this behavior before.
